k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -42,8 +42,8 @@ logsSubscriptionId: ""
 dynatraceLogIngestUrl: ""
 
 activeGate:
-  #useExisting if set to false (by default) the Active Gate will be deployed in k8s. In that case also the activeGate.dynatracePaasToken must be filled.
-  useExisting: "false"
+  #useExisting if set to true (by default) integration will used existing Active Gate, if it set to false the Active Gate will be deployed in k8s. In that case also the activeGate.dynatracePaasToken must be filled.
```

Comment:
imo the message should be similar to the one for Azure/AWS log forwarder (it's important to mention about ingest through the Cluster API):
`If you choose to use direct ingest through the Cluster API or existing ActiveGate, put 'true'.
If you choose new ActiveGate deployment, put 'false'. In such case, ActiveGate will be deployed in k8s...`

Reply:
Good idea, thanks!